content/en/network_monitoring/devices/troubleshooting.md

```diff
@@ -12,91 +12,165 @@ further_reading:
 
 Use the information below for troubleshooting Datadog Network Device Monitoring. If you need additional help, contact [Datadog support][1].
 
-## Terminology
+### Device not visible in Datadog
+
+The following assumes you are running Datadog Agent version `7.59.0+`.
```

Comment:
Per discussion on slack, there are some nicer error messages in 7.61+

```suggestion
The following assumes you are running Datadog Agent version `7.61.0+`.
```

Reply:
Thanks @leeavital ! Per 2nd discussion, we'll keep as is for now and iterate on this and the Autodiscovery section once 7.61 is out.